_sass/_base.scss

```diff
@@ -670,4 +670,25 @@ pre {
   .entry:last-child {
     border-bottom: none;
   }
-}
\ No newline at end of file
+}
+
+.ehcache-documentation {
```

Comment:
If I had done that, I think it would have applied the font size to the whole section under `.ehcache-documentation`, so the whole page would have been impacted.

But I could have done:

````
.ehcache-documentation {

    .future, .current, .old {
        font-size: 14px;
    }

}
````